Sources/StreamChat/StateLayer/Chat.swift

```diff
@@ -880,6 +887,28 @@ public final class Chat {
     public func stopWatching() async throws {
         try await channelUpdater.stopWatching(cid: cid)
     }
+    
+    /// Loads watchers for the specified pagination parameters and updates ``ChatState.watchers``.
```

Comment:
What's the difference between these 2 methods? The "more" part should be handled by the pagination, right?